core-bundle/src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -348,6 +348,7 @@ private function setCacheHeaders(Response $response)
 		/** @var $objPage PageModel */
 		global $objPage;
 
+		// Do not cache response if caching was not configured at all and make sure the response is private.
 		if (($objPage->cache === false || $objPage->cache < 1) && ($objPage->clientCache === false || $objPage->clientCache < 1))
 		{
 			$response->headers->addCacheControlDirective('no-cache');
```

Comment:
I think we should replace this with `$response->headers->set('Cache-Control', 'no-cache, no-store');`

Reply:
Done